lazer/publisher_sdk/proto/governance_instruction.proto

```diff
@@ -321,43 +337,68 @@ message UpdateFeed {
     // [required]
     // Note: when adding a new variant here, update `Permissions` as well.
     oneof action {
-        UpdateFeedMetadata update_feed_metadata = 101;
-        ActivateFeed activate_feed = 102;
-        DeactivateFeed deactivate_feed = 103;
+        UpdateFeedProperties update_feed_properties = 101;
+        UpdateFeedMetadata update_feed_metadata = 102;
+        EnableFeedInShard enable_feed_in_shard = 103;
+        DisableFeedInShard disable_feed_in_shard = 104;
         google.protobuf.Empty remove_feed = 199;
     }
 }
 
+// Update a feed's properties. The feed will be updated with values present in each field.
+// If a value is not supplied, the corresponding property will remain unchanged.
+// Refer to `Feed` message fields documentation.
+message UpdateFeedProperties {
+    // [optional]
+    optional DynamicValue.Map metadata = 3;
+    // [optional]
+    optional string name = 101;
+    // [optional]
+    optional sint32 exponent = 102;
+    // [optional]
+    optional uint32 min_publishers = 103;
+    // [optional]
+    optional google.protobuf.Duration min_rate = 104;
+    // [optional]
+    optional google.protobuf.Duration expiry_time = 105;
+    // [optional]
+    optional string market_schedule = 106;
+    // [optional]
+    optional FeedState state = 107;
+    // [optional]
+    optional bool is_enabled_in_shard = 201;
```

Comment:
I think we should add asset_type here too. Because we will be using asset_type to determine what kind of feed type is expected (e.g funding-rate vs regular price).